src/Searchlight/Exceptions/FieldNotFound.cs

```diff
@@ -19,7 +18,8 @@ public class FieldNotFound : SearchlightException
         public string ErrorMessage
         {
             get =>
-                $"The query filter, {OriginalFilter}, had a field name that was not known. Check the list of known fields to see if the filter contains a typographical error: {KnownFields}";
```

Comment:
Yeah, in retrospect I should have noticed this would print an object rather than a string